<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiblioWindow</class>
 <widget class="QMainWindow" name="BiblioWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ChargeBoutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger la Bibliothèque</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveBoutton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DetailsButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QListWidget" name="AffichageBiblio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="TraitementButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Traitement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRechercherp">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deco">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageCount">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre d'images : 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="triprix">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>triprix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trinbtraitements">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>trinbtrait</string>
    </property>
   </widget>
   <widget class="QPushButton" name="souslistetype">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>souslistetype</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSousListePrix">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>souslisteprix</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/> 
   <widget class="QMenu" name="menuDescripteurs">
    <property name="title">
     <string>Descripteurs</string>
    </property>
    <addaction name="actionAjouterDescripteur"/>
    <addaction name="actionModifierDescripteur"/>
    <addaction name="actionSupprimer_un_descripteur"/>
   </widget>
   <addaction name="menuDescripteurs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjouterDescripteur">
   <property name="text">
    <string>Ajouter un descripteur</string>
   </property>
  </action>
  <action name="actionModifierDescripteur">
   <property name="text">
    <string>Modifier un descripteur</string>
   </property>
  </action>
  <action name="actionSupprimer_un_descripteur">
   <property name="text">
    <string>Supprimer un descripteur</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
